packages/dynamic_color/macos/dynamic_color/Sources/dynamic_color/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array/>
+	<key>NSPrivacyTracking</key>
+	<false/>
+</dict>
+</plist>
```

Comment:
Adding an empty Privacy Manifest is the current solution by all Flutter first-party plugins (even for macOS), see:

* https://github.com/flutter/packages/pull/5846
* https://github.com/flutter/flutter/issues/145269
* https://github.com/flutter/flutter/issues/155564

I added it to CocoaPods integration too.